langwrappers/src/main/kotlin/org/jetbrains/research/testspark/langwrappers/strategies/JavaKotlinFullTextExtractionStrategy.kt

```diff
@@ -0,0 +1,37 @@
+package org.jetbrains.research.testspark.langwrappers.strategies
+
+import com.intellij.psi.PsiFile
+import org.jetbrains.research.testspark.core.utils.importPattern
+import org.jetbrains.research.testspark.core.utils.packagePattern
+
+/**
+Direct implementor for the Java and Kotlin PsiWrappers
+ */
+class JavaKotlinFullTextExtractionStrategy {
```

Comment:
Let's make the name a bit less verbose:
`JavaKotlinFullTextExtractionStrategy` -> `JavaKotlinClassTextExtractor`.